tests/src/FunctionalJavascript/Form/ImporterFormTest.php

```diff
@@ -0,0 +1,205 @@
+  }
+
+  /**
+   * Tests editing an existing importer.
+   */
+  public function testEditImporter() {
+    // Create a migration entity.
+    $migration = $this->createMigration();
+    $migration->set('source', [
+      'plugin' => 'url',
+      'data_fetcher_plugin' => 'file',
+      'data_parser_plugin' => 'json',
+      'item_selector' => '/items/item',
+    ]);
+    $migration->set('destination', [
+      'plugin' => 'entity:node',
+      'default_bundle' => 'article',
+    ]);
+    $migration->save();
+
+    // Create an importer entity.
+    $importer = $this->createImporter();
+    $importer->set('migrationConfig', [
+      'source' => [
+        'data_fetcher_plugin' => 'http',
+        'urls' => [
+          'https://www.news.co.jp/api/articles',
+        ],
+      ],
+      'destination' => [],
+    ]);
+    $importer->save();
+
+    // Navigate to edit our importer.
+    $this->drupalGet("/admin/content/feeds-migrate/importer/{$importer->id()}/edit");
+    $session = $this->assertSession();
+
+    $session->fieldValueEquals('edit-label', $importer->label());
+    $session->fieldValueEquals('edit-importfrequency', $importer->getImportFrequency());
+    $session->fieldValueEquals('existing', $importer->getExisting());
+    $session->fieldValueEquals('edit-keeporphans', $importer->keepOrphans());
+    $session->fieldValueEquals('edit-migrationid', $importer->getMigrationId());
+    $session->fieldValueEquals('migration[source][plugin]', $importer->getMigration()->get('source')['plugin']);
+    $session->fieldValueEquals('migration[source][data_fetcher_plugin]', $importer->getMigration()->get('source')['data_fetcher_plugin']);
+    $session->fieldValueEquals('source_wrapper[importer][data_fetcher_wrapper][importer][urls]', $importer->getMigration()->get('source')['urls'][0]);
+    $session->fieldValueEquals('migration[source][data_parser_plugin]', $importer->getMigration()->get('source')['data_parser_plugin']);
+    $session->fieldValueEquals('migration[destination][plugin]', $importer->getMigration()->get('destination')['plugin']);
+    $session->fieldValueEquals('destination_wrapper[options][default_bundle]', $importer->getMigration()->get('destination')['default_bundle']);
+  }
+
+  /**
+   * Tests deleting an importer.
+   */
+  public function testDeleteImporter() {
+    // Create a migration entity.
+    $migration = $this->createMigration();
+    $migration->set('source', [
+      'plugin' => 'url',
+      'data_fetcher_plugin' => 'file',
+      'data_parser_plugin' => 'json',
+      'item_selector' => '/items/item',
+    ]);
+    $migration->set('destination', [
+      'plugin' => 'entity:node',
+      'default_bundle' => 'article',
+    ]);
+    $migration->save();
+
+    // Create an importer entity.
+    $importer = $this->createImporter();
+    $importer->set('migrationConfig', [
+      'source' => [
+        'data_fetcher_plugin' => 'http',
+        'urls' => [
+          'https://www.news.co.jp/api/articles',
```

Comment:
As said earlier, I think it's better to use www.example.com instead of real looking urls.